apps/web/src/utils/shouldShowSwapWarning.ts

```diff
@@ -1,8 +1,8 @@
 import { Token } from '@pancakeswap/sdk'
 import SwapWarningTokens from 'config/constants/swapWarningTokens'
 
-const shouldShowSwapWarning = (chainId: number, swapCurrency: Token): boolean => {
-  if (SwapWarningTokens[chainId]) {
+const shouldShowSwapWarning = (swapCurrency: Token, chainId?: number): boolean => {
```

Comment:
we better not change the function signature if just for optional type. I think it's ok to specify undefined explicitly. Also let's use `ChainId` type if possible
```suggestion
const shouldShowSwapWarning = (chainId: ChainId | undefined, swapCurrency: Token): boolean => {
```